packages/wdio-browserstack-service/src/service.ts

```diff
@@ -48,13 +49,19 @@ export default class BrowserstackService implements Services.ServiceInstance {
         this._config || (this._config = this._options)
         this._observability = this._options.testObservability
         this._accessibility = this._options.accessibility
+        this._turboScale = this._options.turboScale
 
         if (this._observability) {
             this._config.reporters?.push(TestReporter)
             if (process.env.BROWSERSTACK_O11Y_PERF_MEASUREMENT) {
                 PerformanceTester.startMonitoring('performance-report-service.csv')
             }
         }
+
+        if (process.env.BROWSERSTACK_TURBOSCALE) {
+            this._turboScale = true
```

Comment:
Why are both the options and the environment variable used to set the _turboScale property?

Reply:
Two behaviours, user can either pass `turboScale: true/false` in service options as well as just by setting up the env var.
We want env var to override the one set in config.

Reply:
Just to correct, I will be updating the above statement (L62) to `this._turboScale = process.env.BROWSERSTACK_TURBOSCALE === 'true'`

Reply:
The environment variable is a requirement by Browserstack is it not? If that is true, I think it makes sense that we set the variable for the user when the option is passed through the options object instead

Reply:
@erwinheitzman It's not mandatory. TurboScale can be set to true by either setting the environment variable or by setting `turboScale` key in options in config. So if any one of them is set, we use that.

Reply:
@erwinheitzman did you get a chance to see the above response? Let me know if it clears your question or not.